Controller/MediaController.php

```diff
@@ -106,6 +108,8 @@ public function viewAction($id, $format = 'reference')
      * @param string $filter
      *
      * @return Response
+     *
+     * @deprecated since 3.x, to be removed in 4.0.
```

Comment:
There should probably be an `E_USER_DEPRECATED` error, indicating what the new way of doing things is.